ui/app/mirrors/create/tablemapping.tsx

```diff
@@ -116,11 +116,10 @@ const TableMapping = ({
             return table.toLowerCase().includes(searchQuery.toLowerCase());
           })
       );
-    }
-    if (searchQuery.length == 0) {
+    } else if (searchQuery.length == 0) {
       getTablesOfSchema(schema);
     }
-  }, [searchQuery, getTablesOfSchema]);
+  }, [searchQuery, getTablesOfSchema, schema]);
```

Comment:
Unfortunately, adding `schema` to the useEffect in the Search component results in this infinite rerender issue in the `/mirrors/edit/<mirror_name>` page.
Maybe I should use an onchange function instead of useEffect here
<img width="327" alt="Screenshot 2023-11-06 at 11 49 48 PM" src="https://github.com/PeerDB-io/peerdb/assets/65964360/4403920e-7649-416e-a862-64dd56808f51">

Reply:
I worked a fix for this here: https://github.com/PeerDB-io/peerdb/pull/621

Reply:
Great, left some feedback, will remove these files from this PR